packages/gatsby-theme-newrelic/src/components/Button.js

```diff
@@ -45,6 +49,14 @@ const styles = {
         color: var(--link-hover-color);
       }
     `,
+    [VARIANTS.LOCALE]: css`
```

Comment:
I'm curious, what is a "locale" button? Is this only for the global header? If so, its worth considering making this a one-off there instead of a global type. Then again I have no context for these changes, so feel free to ignore me 😄

Reply:
The "locale" button is intended for the dropdown section in the header for language preference. Assuming that we are going to be adding more languages in the future, there may be multiple options where adding a  "locale" button variant may be beneficial. 

I'm not particularly attached to this idea, so I'm open to suggestions and happy to change this.